.gitignore

```diff
@@ -9,9 +9,38 @@ go-test-tmpfile*
 coverage.txt
 .idea/
 .DS_Store
+build/
+release/
```

Comment:
only these 2 additions make sense to me, the rest was never there and this seems not to have caused problems, so please remove those.